node/pkg/logscribeconsumer/app.go

```diff
@@ -131,29 +163,33 @@ func isLogLevelValid(entry map[string]any) bool {
 	return true
 }
 
-func bulkCopyLogEntries(ctx context.Context, logEntries []map[string]any) error {
-	bulkCopyEntries := [][]any{}
+func (a *App) bulkPostLogEntries(logEntries []map[string]any) error {
+	bulkCopyEntries := []LogInsertModel{}
```

Comment:
let's rename variable name too, it's not bulkCopying anymore so maybe `bulkLogInsertEntries`